spanner/cloud-client/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -69,9 +71,21 @@ public static void setUp() throws Exception {
     spanner = options.getService();
     dbClient = spanner.getDatabaseAdminClient();
     dbId = DatabaseId.of(options.getProjectId(), instanceId, databaseId);
-    dbClient.dropDatabase(dbId.getInstanceId().getInstance(), dbId.getDatabase());
-    dbClient.dropDatabase(
-        dbId.getInstanceId().getInstance(), SpannerSample.createRestoredSampleDbId(dbId));
+    // Delete stale test databases that have been created earlier by this test, but not deleted.
+    deleteStaleTestDatabases(instanceId, System.getProperty("spanner.sample.database"));
```

Comment:
Can we reuse `baseDbId` here instead of the property again?

Reply:
Good point, fixed.